Ginger/GingerCoreCommon/VariablesLib/VariableSelectionList.cs

```diff
@@ -41,12 +41,42 @@ public override string VariableType
             get { return "Selection List"; }
         }
 
+        [IsSerializedForLocalRepository(true)]        
```

Comment:
Please remove [IsSerializedForLocalRepository(true)] for mIsLoopEnabled as we have same attribute for IsLoopEnabled